### PR TITLE
[16.0][IMP] edi_product_oca: Plug EDI framework to product.packaging

### PR DESCRIPTION
**Pull request by QuocDuong1306**

### CODE REVIEW

File: edi_product_oca/__manifest__.py [hunk 1]
```diff
@@ -4,14 +4,20 @@
...
+       EDI framework configuration and base logic for products, product packagings
+       and product barcodes.
+       """,
     "version": "16.0.1.0.0",
     "license": "AGPL-3",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/edi-framework",
     "depends": [
         "product",
+        "product_multi_barcode",
```
Comment (simahawk) [inline edi_product_oca/__manifest__.py]: this should go to its own module (eg: edi_product_multi_barcode) w/ auto_install=True

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
FTR it seems there's conflict in tests when using endpoint https://github.com/OCA/edi-framework/pull/46#issuecomment-1859986272

File: edi_product_oca/__manifest__.py (hunk 1 above)
Comment (QuocDuong1306) [reply edi_product_oca/__manifest__.py]: Thank @simahawk , I have created a PR for new [edi_product_multi_barcode_oca](https://github.com/OCA/edi-framework/pull/54)

**Review by simahawk — APPROVED** [review]

**Pull request merged**